app/models/user.rb

```diff
@@ -1,10 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please also add confirmable as the requirement say and verify that the token is generated in the terminal before sending it before review.

Reply:
Thanks for catching that! I fixed it